src/components/matrix/Matrix.java

```diff
@@ -0,0 +1,117 @@
+package src.components.matrix;
+
+//Hello Jeremy
+
+public class Matrix {// extends components.standard<Matrix> {
+    private int[][] rep;
+
+    private void createNewRep(int i, int j) {
+        this.rep = new int[i][j];
+
+        for (int x = 0; x < i; x++) {
+            for (int y = 0; y < j; y++) {
+                this.rep[x][y] = 0;
+            }
+        }
+    }
+
+    public Matrix(int[][] m) {
```

Comment:
This is great! If you want, you can also have a constructor for a "serialized" array. Something like:

```java
public Matrix(int[] data, int rows, int cols)
```

I had to do something like this before since command line inputs don't really hand 2D data well.

Reply:
I suppose `cols` is redundant here since you can divide the length by rows to figure out cols.

Reply:
Also, creating arrays is somewhat painful in Java. I might even make a constructor that looks like this:

```java
public Matrix(int rows, int... data)
```

That way, the client doesn't even have to make an array. They can just list a bunch of numbers.